src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Services/Endpoints/Payments/ListPaymentsEndpoint.cs

```diff
@@ -0,0 +1,38 @@
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+using TicketManager.PaymentService.Contracts.Payments;
+using TicketManager.PaymentService.Services.DataAccess;
+using TicketManager.PaymentService.Services.DataAccess.DtoMappers;
+
+namespace TicketManager.PaymentService.Services.Endpoints.Payments;
+
+public class ListPaymentsEndpoint: EndpointWithoutRequest<ListPaymentsResponse>
```

Comment:
Why does this endpoint even exist? For debug purposes or what?

Reply:
Yes, debug purpouses only